### PR TITLE
Update getting started guide links to helm charts

### PR DESCRIPTION
**Pull request by maxgio92**
### What does this PR do?

Fixes links to Helm chart in the Getting started documentation.

### Motivation

The links to Helm charts were broken.

### CODE REVIEW

File: docs/getting_started.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Using the Datadog Operator requires the following prerequisites:
 
 ## Deploy an agent with the operator
```
Comment (ruthnaebeck) [inline docs/getting_started.md:14]:
```suggestion
## Deploy the Agent with the operator
```

File: docs/getting_started.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ Using the Datadog Operator requires the following prerequisites:
...
 
-In order to deploy a Datadog agent with the operator in the minimum number of steps, the [`datadog-agent-with-operator`](https://github.com/DataDog/datadog-operator/tree/master/chart/datadog-agent-with-operator) helm chart can be used.
+In order to deploy a Datadog agent with the operator in the minimum number of steps, the [`datadog-operator`](https://github.com/DataDog/helm-charts/tree/master/charts/datadog-operator) helm chart can be used.
```
Comment (ruthnaebeck) [inline docs/getting_started.md:16]:
```suggestion
To deploy the Datadog Agent with the operator in the minimum number of steps, see the [`datadog-operator`](https://github.com/DataDog/helm-charts/tree/master/charts/datadog-operator) helm chart.
```

**Review by ruthnaebeck — CHANGES_REQUESTED** [review]
Docs review

**Comment by maxgio92** [comment]
> Docs review

Thank you. Applied.

**Review by clamoriniere — APPROVED** [review]

**Pull request merged**